app/client/src/components/pages/State/lookups/introText.js

```diff
@@ -697,24 +697,25 @@ export default {
   },
   AR: {
     region: 6,
-    intro: null,
+    intro:
+      'The Arkansas Department of Energy and Environment, Division of Environmental Quality (DEQ), Office of Water Quality (OWQ) implements the Federal Clean Water Act through its water quality standards, surface water monitoring, 303(d), 305(b), TMDL, NPDES, 401 Certification, storm water permitting, compliance, and enforcement programs. The Water Quality Planning Branch (Planning) oversees water quality standards, monitoring, assessments, listing, reporting, and TMDL development programs. With the aid of the Compliance Branch, Planning monitors surface waters across the state with its ambient water quality network, ambient lake sampling program, and special projects. These monitoring data make up the bulk of Arkansas’s assessment database. However, the State routinely receives additional data from other entities including Universities, watershed groups, and state and federal agencies. Arkansas’s nonpoint, 319, program is implemented by DEQ’s sister agency Arkansas Natural Resources Commission (ANRC). Planning “special projects” include, but are not limited to, criteria development, use attainability analyses, mining reclamation, focused impairment issues, karst geology, and harmful algal blooms (HABS.',
```

Comment:
Just need to add a closing parentheses after HABS

```suggestion
      'The Arkansas Department of Energy and Environment, Division of Environmental Quality (DEQ), Office of Water Quality (OWQ) implements the Federal Clean Water Act through its water quality standards, surface water monitoring, 303(d), 305(b), TMDL, NPDES, 401 Certification, storm water permitting, compliance, and enforcement programs. The Water Quality Planning Branch (Planning) oversees water quality standards, monitoring, assessments, listing, reporting, and TMDL development programs. With the aid of the Compliance Branch, Planning monitors surface waters across the state with its ambient water quality network, ambient lake sampling program, and special projects. These monitoring data make up the bulk of Arkansas’s assessment database. However, the State routinely receives additional data from other entities including Universities, watershed groups, and state and federal agencies. Arkansas’s nonpoint, 319, program is implemented by DEQ’s sister agency Arkansas Natural Resources Commission (ANRC). Planning “special projects” include, but are not limited to, criteria development, use attainability analyses, mining reclamation, focused impairment issues, karst geology, and harmful algal blooms (HABS).',
```

Reply:
Thanks @cschwinderg! I'll fix that typo and double check the Wisconsin metrics with Kiki

Reply:
Kiki confirmed the Assessed Waters value should be 30% and the Arkansas typo is now corrected

Reply:
Thank you for making the change and checking up on that.